src/app/home/home.component.html

```diff
@@ -29,10 +29,10 @@
     <p class="menu-item-text" *ngIf="notification.doc.status==='unread' && notification.doc.link ===''">
       {{notification.doc.message}} {{notification.doc.time | date: 'MMM d, yyyy'}}
     </p>
-    <p *ngIf="notification.status!=='unread' && notification.doc.link">

```

Comment:
These changes from the `Duplicate notification` commit should be removed as they are for a different PR.  Please make sure to begin branches from the `master` branch.